AUTHORS

```diff
@@ -0,0 +1,3 @@
+See root_files/humans.txt
+
+Please add yourself when you contribute code if you like.
```

Comment:
Should we not perhaps rephrase this to be something like:

    "Software is made by humans, so if you have contributed to Bedrock, please add your Github username to the
      above mentioned [humans.txt][] file. Thank you for your contribution.

    [humans.txt]: https://github.com/mozilla/bedrock/blob/master/root_files/humans.txt"